helpers/guilds/getBan.ts

```diff
@@ -1,16 +1,22 @@
 import type { Bot } from "../../bot.ts";
+import { User } from "../../transformers/member.ts";
 import { DiscordBan } from "../../types/discord.ts";
 
+export type Ban = {
+  reason: string | undefined;
```

Comment:
```suggestion
  reason?: string;
```

Reply:
I believe I used `undefined` here because the TS analyser had issues with inferring the type in `getBans.ts`. Updated in next commit.